apidoc/Titanium/UI/ShortcutItem.yml

```diff
@@ -0,0 +1,77 @@
+---
+name: Titanium.UI.ShortcutItem
+summary: An application shortcut.
+description: |
+    Allows the creation of application shortcuts, which can be detected using
+    the `shortcutitemclick` event from <Titanium.App>.
+
+    Use the <Titanium.UI.createShortcutItem> method to create a shortcut.
+
+extends: Titanium.Proxy
+since: "7.1.0"
+methods:
+
+  - name: show
+    description: Allow the shortcut to show.
+    platforms: [android, iphone, ipad]
+
+  - name: hide
+    description: Hide the shortcut.
+    platforms: [android, iphone, ipad]
+
+properties:
+  - name: id
+    summary: Determines shortcut id.
+    description: This should be a unique identifier for the shortcut.
+    type: String
+    platforms: [android, iphone, ipad]
+    availability: creation
+
+  - name: title
+    description: Title of the shortcut.
+    type: String
+    platforms: [android, iphone, ipad]
+    availability: creation
+
+  - name: subtitle
+    description: Subtitle of the shortcut.
+    type: String
+    platforms: [android, iphone, ipad]
+    availability: creation
+
+  - name: icon
+    summary: Shortcut icon.
+    description: This can be set as a resource id or file path.
+    type: [String, Number]
```

Comment:
We should add iOS-specific notes here, because it can be a Ti.Contacts.Person instance, a constant (`SHORTCUT_ICON_TYPE_*`) or name of an image (e.g. `settings` for an image named `settings.png`).